server/src/main/java/org/elasticsearch/indices/flush/SyncedFlushService.java

```diff
@@ -387,9 +387,9 @@ void sendSyncRequests(final String syncId,
             if (preSyncedResponse.numDocs != numDocsOnPrimary &&
                 preSyncedResponse.numDocs != PreSyncedFlushResponse.UNKNOWN_NUM_DOCS &&
                 numDocsOnPrimary != PreSyncedFlushResponse.UNKNOWN_NUM_DOCS) {
-                logger.warn("{} can't to issue sync id [{}] for out of sync replica [{}] with num docs [{}]; num docs on primary [{}]",
+                logger.trace("{} can't to issue sync id [{}] for replica [{}] with num docs [{}]; num docs on primary [{}]",
```

Comment:
Minor wording nit and I think we could reasonably log this at `debug` since it's still something of a failure.

```suggestion
                logger.debug("{} can't issue sync id [{}] for replica [{}] with num docs [{}]; num docs on primary [{}]",
```